agent/handlers/taskmetadata/handler.go

```diff
@@ -40,6 +41,9 @@ const (
 	// Credentials API versions
 	apiVersion1 = 1
 	apiVersion2 = 2
+	// RequestThrottleStatusCode is the http response code when the request exceeds
+	// the request limit
+	RequestThrottleStatusCode = 429
```

Comment:
You shouldn't need to define this. Just use `http.StatusTooManyRequests` instead